modern/src/common/style.js

```diff
@@ -0,0 +1 @@
+export const columnWidthNormal = 100;
```

Comment:
No, I don't like this solution. We should figure out a way without hardcoded width.